website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -50,6 +50,7 @@ and further information.
 [olm-docs]: /docs/olm-integration/
 [ocp-route]: https://docs.openshift.com/container-platform/4.9/rest_api/network_apis/route-route-openshift-io-v1.html#route-route-openshift-io-v1
 [ocp-ingress-doc]: https://docs.openshift.com/container-platform/4.9/networking/ingress-operator.html#nw-ingress-sharding-route-labels_configuring-ingress
+[k8s-ingress-controllers]: https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/
```

Comment:
@camilamacedo86 Just wanted to check where is `k8s-ingress-controllers` being referenced? I couldn't find it in the doc, maybe its me missing it!

Reply:
See: https://github.com/operator-framework/operator-sdk/blob/699f27b052f7fe6dbcd05e343d0ecd5b5c8c6f60/website/content/en/docs/best-practices/multi-tenancy.md?plain=1#L27-L35

Reply:
<img width="1109" alt="Screenshot 2021-12-12 at 13 27 50" src="https://user-images.githubusercontent.com/7708031/145714239-4db0f552-39f6-4d12-adcc-fdab513f3a76.png">